Update/kage_008.txt

```diff
@@ -649,7 +649,7 @@ void main()
 //rそういえば、ホテルのベッドから起きた時に身体がやや重ったるい気分だった。…もっとも、それは私の鬱屈した気持ちがそう感じさせているだけかもしれないが。
 	if (GetGlobalFlag(GADVMode)) { OutputLineAll("", NULL, Line_ContinueAfterTyping); }
 	OutputLine(NULL, "そういえば、ホテルのベッドから起きた時に身体がやや重ったるい気分だった。…もっとも、それは私の鬱屈した気持ちがそう感じさせているだけかもしれないが。",
-			NULL, "Come to think of it, my body felt a little heavy when I got up from the hotel bed. ...Of course, that might just be because I've been feeling depressed lately.", Line_Normal);
+			NULL, "Come to think of it, my body felt a little heavy when I got up from the hotel bed. ...Of course, that might just be my depression making me feel that way.", Line_Normal);
```

Comment:
```suggestion
			NULL, "Come to think of it, my body felt a little heavy when I got up from the hotel bed. ...Of course, that might just be because I've been feeling depressed lately.", Line_Normal);
```